documentation/install_from_packages.markdown

```diff
@@ -50,34 +65,7 @@ There is no `-` in the package name.
 systemctl start puppetserver
```

Comment:
Most users will probably know/figure this out, but I wonder if it's worth adding `sudo` to these commands?

Reply:
I think it would be good to add a command to the end, just for users to check that it's working. Maybe just `puppetserver -v` to check the version?